api-catalog-ui/frontend/package.json

```diff
@@ -36,13 +36,13 @@
         "redux-thunk": "2.3.0",
         "rimraf": "3.0.2",
         "rxjs": "6.3.3",
-        "swagger-ui": "3.25.0",
+        "swagger-ui-react": "^3.23.0",
```

Comment:
Could you please update the `swagger-ui-react` version to the latest one? It's `3.25.0`. Also please get rid of `^` so it will point to the specific version.